src/Authentication/Authentication.Core/ErrorConstants.cs

```diff
@@ -15,6 +15,7 @@ internal static class Codes
             internal const string SessionLockWriteDisposed = "sessionLockWriteDisposed";
             internal const string SessionLockWriteRecursion = "sessionLockWriteRecursion";
             internal const string InvalidJWT = "invalidJWT";
+            internal const string OutputNotInitialized = nameof(OutputNotInitialized);
```

Comment:
seems there's a pattern of naming with lower case.  should stay consistent.

Reply:
I would propose we take advantage of `nameof` and make the rest upper case?

Reply:
could someone have taken a dependency on the existing strings?  Example, they look for the a specific error message including the string and didn't handle case sensitivity.

Reply:
+1 to what @ddyett said.

@georgend Having error codes in camelCase is also standard even for our [.NET SDK](https://github.com/microsoftgraph/msgraph-sdk-dotnet-core/blob/dev/src/Microsoft.Graph.Core/Exceptions/ErrorConstants.cs#L9). We do use camelCasing in PowerShell to align with the casing in .NET Core SDK since we do take a dependency on it and have cases where we pass-through exceptions to PowerShell's error stream.

Reply:
@peombwa Thanks for the pointer.